packages/@webex/plugin-cc/test/unit/spec/cc.ts

```diff
@@ -86,6 +89,7 @@ describe('webex.cc', () => {
 
   afterEach(() => {
     jest.clearAllMocks();
+    eventBusInstance.removeAllListeners();
```

Comment:
In order for any remaining event listeners to be removed (otherwise memory used is intensive).